play-scala-tls-example/play

```diff
@@ -22,40 +22,34 @@ JVM_OPTIONS="$JVM_OPTIONS -Dplay.http.sslengineprovider=https.CustomSSLEnginePro
 #JVM_OPTIONS="$JVM_OPTIONS -Dplay.ssl.needClientAuth=true"
 
 # Enable the handshake parameter to be extended for better protection.
-# http://docs.oracle.com/javase/8/docs/technotes/guides/security/jsse/JSSERefGuide.html#customizing_dh_keys
+# https://docs.oracle.com/en/java/javase/11/security/java-secure-socket-extension-jsse-reference-guide.html#GUID-D9B216E8-3EFC-4882-B76E-17A87D8F2F9D
 # Only relevant for "DHE_RSA", "DHE_DSS", "DH_ANON" algorithms, in ServerHandshaker.java.
 JVM_OPTIONS="$JVM_OPTIONS -Djdk.tls.ephemeralDHKeySize=2048"
 
 # Don't allow client to dictate terms - this can also be used for DoS attacks.
-# Undocumented, defined in sun.security.ssl.Handshaker.java:205
+# Undocumented, defined in https://github.com/openjdk/jdk11u/blob/jdk-11.0.20%2B1/src/java.base/share/classes/sun/security/ssl/ServerHandshakeContext.java#L35-L43
 JVM_OPTIONS="$JVM_OPTIONS -Djdk.tls.rejectClientInitiatedRenegotiation=true"
 
 # Add more details to the disabled algorithms list
-# http://docs.oracle.com/javase/8/docs/technotes/guides/security/jsse/JSSERefGuide.html#DisabledAlgorithms
-# and http://bugs.java.com/bugdatabase/view_bug.do?bug_id=7133344
+# https://docs.oracle.com/en/java/javase/11/security/java-secure-socket-extension-jsse-reference-guide.html#GUID-0A438179-32A7-4900-A81C-29E3073E1E90
+# and https://bugs.java.com/bugdatabase/view_bug.do?bug_id=7133344
 JVM_OPTIONS="$JVM_OPTIONS -Djava.security.properties=disabledAlgorithms.properties"
 
 # Fix a version number problem in SSLv3 and TLS version 1.0.
-# http://docs.oracle.com/javase/7/docs/technotes/guides/security/SunProviders.html
+# https://docs.oracle.com/en/java/javase/11/security/oracle-providers.html
 JVM_OPTIONS="$JVM_OPTIONS -Dcom.sun.net.ssl.rsaPreMasterSecretFix=true"
 
-# Tighten the TLS negotiation issue.
-# http://docs.oracle.com/javase/8/docs/technotes/guides/security/jsse/JSSERefGuide.html#descPhase2
-# Defined in JDK 1.8 sun.security.ssl.Handshaker.java:194
-JVM_OPTIONS="$JVM_OPTIONS -Dsun.security.ssl.allowUnsafeRenegotiation=false"
-JVM_OPTIONS="$JVM_OPTIONS -Dsun.security.ssl.allowLegacyHelloMessages=false"
-
```

Comment:
This isn't needed anymore since Java 11 is required now and contains the fixes necessary. This was added in https://github.com/playframework/play-samples/commit/abe0e58888ffcfd79ed5e2343593e6df5dd6bb91 where Java 7 was still in use and also the first Java 8 release still didn't fix the problem.

See
* Java 7: https://docs.oracle.com/javase/7/docs/technotes/guides/security/jsse/JSSERefGuide.html#descPhase2
* Java 8: https://docs.oracle.com/javase/8/docs/technotes/guides/security/jsse/JSSERefGuide.html#tls-renegotiation-issue
* https://docs.oracle.com/javase/8/docs/technotes/guides/security/jsse/tls-renegotiation-issue.html
* In Java 11 the flags are already deprecated: https://docs.oracle.com/en/java/javase/11/security/java-secure-socket-extension-jsse-reference-guide.html#GUID-460C3E5A-A373-4742-9E84-EB42A7A3C363